portfolio-style.css

```diff
@@ -0,0 +1,258 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    width: 100%;
+    overflow: auto;
+    border: 1px solid #FF0FA8;
+}
+
+.top-bar{
+    overflow: auto;
+    font-family:  'Lucida Console', Monaco, monospace;
+    background-color: #EEC643
+}
+
+.top-stuff  {
+    position: relative;
+}
+
+
+.name {
+    float: left;
+    margin: 0px 10px;
+    font-family: Palatino;
+    color: #EAF2E3;
+    font-size: 1px;
+}
+
+h3 {
+font-size: 26px;
+}
+
+
+.hero{
+    z-index: 10;
+    top: -600px;
+    padding: 0px 20px;
+    color: #ffffff;
+    font-family: Verdana;
+    position: relative;
+    font-size: 32px;
+}
+
+.squire {
+    z-index: 10;
+    top: -640px;
+    padding: 0px 85px;
+    color: #ffffff;
+    font-family: Tahoma, Geneva, sans-serif;
+    position: relative;
+    font-size: 22px;
+
+}
+
+
+.top-links{
+    float: right;
+}
+
+.top-links button {
```

Comment:
No media query to deal with this, not responsive because buttons collide and wrap, should follow design that was signed off by PO